src/Docker/Runner/VariablesContext.php

```diff
@@ -2,14 +2,37 @@
 
 namespace Keboola\DockerBundle\Docker\Runner;
 
-class VariablesContext extends \ArrayObject
+class VariablesContext
 {
+    private $missingVariables;
+    private $values;
+
     public function __construct(array $configurationRow)
     {
-        $values = [];
+        $this->values = [];
         foreach ($configurationRow['values'] as $row) {
-            $values[$row['name']] = $row['value'];
+            $this->values[$row['name']] = $row['value'];
         }
-        parent::__construct($values);
+        $this->missingVariables = [];
+    }
+
+    public function __isset($name)
+    {
+        return true;
```

Comment:
Tu vzdy vratis `true`?

Reply:
🤦  wtf, tvl jak je teda mozny, ze me to fungovalo v testech?

Reply:
uz vim, protoze tohle se nikde netestuje, no nic jdu to fixnout

Reply:
Tiez ma to zarazilo :) Ze predsa ten prvy test prechadza...

Reply:
no toto nastane jen pokud je nadefinovana promena, nema zadanou hodnotu a **neni nikde pouzita**, protoze pokud je nekde pouzita, tak to chytne ta kontrola ve variables kontextu. 

Ono je trosku otazka jestli takovou vec vyhazovat jako vyjimku, ale zatim bych to tak nechal a uvidime jak se to osvedci.